src/main/resources/import.sql

```diff
@@ -35,17 +34,17 @@ INSERT INTO Review (rating, description, approved, date, type) VALUES (5, 'Beaut
-INSERT INTO Property (name, min_guests, max_guests, description, latitude, longitude, approved, pricing_mode, cancellation_deadline, auto_approve_enabled, street, city, country) VALUES ('Beach House', 8, 12, 'Relaxing beachfront property', 25.7617, -80.1918, true, 0, 14, true, 'Trg Dositeja Obradovica 6', 'Novi Sad', 'Serbia');
-INSERT INTO Property (name, min_guests, max_guests, description, latitude, longitude, approved, pricing_mode, cancellation_deadline, auto_approve_enabled, street, city, country) VALUES ('Luxury Villa', 10, 20, 'Opulent villa with all amenities', 41.8781, -87.6298, false, 0, 14, false, 'Trg Dositeja Obradovica 6', 'Novi Sad', 'Serbia');
-INSERT INTO Property (name, min_guests, max_guests, description, latitude, longitude, approved, pricing_mode, cancellation_deadline, auto_approve_enabled, street, city, country) VALUES ('Forest Hut', 2, 3, 'Cozy hut in the middle of the forest', 45.5051, -122.6750, true, 0, 7, true, 'Trg Dositeja Obradovica 6', 'Novi Sad', 'Serbia');
-INSERT INTO Property (name, min_guests, max_guests, description, latitude, longitude, approved, pricing_mode, cancellation_deadline, auto_approve_enabled, street, city, country) VALUES ('Riverside Lodge', 6, 8, 'Rustic lodge by the river', 33.7490, -84.3880, true, 0, 21, false, 'Trg Dositeja Obradovica 6', 'Novi Sad', 'Serbia');
-INSERT INTO Property (name, min_guests, max_guests, description, latitude, longitude, approved, pricing_mode, cancellation_deadline, auto_approve_enabled, street, city, country) VALUES ('Country House', 12, 15, 'Charming country house with a garden', 39.9526, -75.1652, true, 0, 30, true, '1510 Market St, Philadelphia, PA 19102', 'Philadelphia PA', 'United States');
-INSERT INTO Property (name, min_guests, max_guests, description, latitude, longitude, approved, pricing_mode, cancellation_deadline, auto_approve_enabled, street, city, country) VALUES ('Skyline Penthouse', 2, 4, 'Elegant penthouse with city views', 37.7749, -122.4194, true, 0, 14, false, 'Trg Dositeja Obradovica 6', 'Novi Sad', 'Serbia');
+INSERT INTO Property (id, name, min_guests, max_guests, description, latitude, longitude, approved, pricing_mode, cancellation_deadline, auto_approve_enabled, street, city, country, host_Id) VALUES (1, 'Shilage Castle', 2, 4, 'Shilage Castle is a castle located in eastern Erusea, within the state of Shilage. Shilage Castle is based on Castle Szigliget, with the landscape surrounding it resembling the Szigliget village. Extract from Acepedia.', 48.9994708,20.7649368, false, 0, 7, true, 'Shilage Castle', 'Shilage', 'Erusea', 10);
+INSERT INTO Property (id, name, min_guests, max_guests, description, latitude, longitude, approved, pricing_mode, cancellation_deadline, auto_approve_enabled, street, city, country, host_Id) VALUES (2, 'Cozy Cabin', 2, 4, 'A charming cabin in the woods', 40.7128, -74.0060, false, 0, 7, true, 'New York City Hall, New York, NY 10007', 'New York NY', 'United States', 10);
```

Comment:
The id in `host_id` starts with an uppercase I in these two rows.
```suggestion
INSERT INTO Property (id, name, min_guests, max_guests, description, latitude, longitude, approved, pricing_mode, cancellation_deadline, auto_approve_enabled, street, city, country, host_id) VALUES (1, 'Shilage Castle', 2, 4, 'Shilage Castle is a castle located in eastern Erusea, within the state of Shilage. Shilage Castle is based on Castle Szigliget, with the landscape surrounding it resembling the Szigliget village. Extract from Acepedia.', 48.9994708,20.7649368, false, 0, 7, true, 'Shilage Castle', 'Shilage', 'Erusea', 10);
INSERT INTO Property (id, name, min_guests, max_guests, description, latitude, longitude, approved, pricing_mode, cancellation_deadline, auto_approve_enabled, street, city, country, host_id) VALUES (2, 'Cozy Cabin', 2, 4, 'A charming cabin in the woods', 40.7128, -74.0060, false, 0, 7, true, 'New York City Hall, New York, NY 10007', 'New York NY', 'United States', 10);
```